msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/ltw/TestCase2582294.java

```diff
@@ -0,0 +1,125 @@
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.broker.ltw;
+
+import com.microsoft.identity.client.msal.automationapp.R;
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalBrokerTest;
+import com.microsoft.identity.client.ui.automation.annotations.LTWTests;
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost;
+import com.microsoft.identity.client.ui.automation.broker.BrokerLTW;
+import com.microsoft.identity.client.ui.automation.broker.BrokerMicrosoftAuthenticator;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+// If LTW is the active broker, and request is made through Authenticator from Legacy WorkplaceJoin API, nothing should break
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2582294
+@SupportedBrokers(brokers = {BrokerLTW.class})
+@LTWTests
+public class TestCase2582294 extends AbstractMsalBrokerTest {
+
+    @Test
+    public void test_2582294() throws Throwable{
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+        final String homeTenantId = mLabAccount.getHomeTenantId();
+
+        // Install new Authenticator with broker SDK changes of broker selection logic
+        final BrokerMicrosoftAuthenticator brokerMicrosoftAuthenticator = new BrokerMicrosoftAuthenticator();
+        brokerMicrosoftAuthenticator.install();
+
+        // Install old BrokerHost app with no broker SDK changes of broker selection logic
+        final BrokerHost brokerHost = new BrokerHost(BrokerHost.BROKER_HOST_WITHOUT_BROKER_SELECTION_APK);
+        brokerHost.install();
+        brokerHost.launch();
+
+        // Enter tenantId in tenantId text box
+        // Click on "Get preprovisioning blob" button
+        // You should see a popup with the blob
+        final String blob = brokerHost.getBlob(homeTenantId);
+        Assert.assertTrue(!blob.isEmpty());
```

Comment:
is there any specific pattern we could assert in the blob? Instead of just checking if the blob is empty?